sparql11/data-sparql11/functions/manifest.ttl

```diff
@@ -733,6 +733,18 @@
     mf:result  <uuid01.srx> ;
 	.
 
+:uuid02 rdf:type mf:QueryEvaluationTest ;
```

Comment:
There isn't a "uuid02" in the  mf:entries list of tests.

Also and maybe related - I could not apply the manifest part as a patch.

Reply:
Added to the entries list. I'm not sure why the patch would have failed.